lib/puppet/util/windows/process.rb

```diff
@@ -60,6 +64,26 @@ def get_current_process
   end
   module_function :get_current_process
 
+  def open_process(desired_access, inherit_handle, process_id, &block)
+    phandle = nil
+    inherit = inherit_handle ? FFI::WIN32_TRUE : FFI::WIN32_FALSE
+    begin
```

Comment:
You should be able to use the implicit `begin` here like the example I wrote up, but not worth holding up PR for - https://github.com/rubocop-hq/ruby-style-guide#begin-implicit